impl/maven-impl/src/main/java/org/apache/maven/impl/model/DefaultModelProcessor.java

```diff
@@ -94,50 +97,45 @@ public Path locateExistingPom(Path projectDirectory) {
         return pom;
     }
 
+    private Path doLocateExistingPom(Path project) {
```

Comment:
no need to use "do" on methods like this. locateExistingPom is still a verb and more natural.

Reply:
yes sorry i expected this form main we on the pretty same page thats outdates somehow sorry.


Can i pitch that we switch to main BTW? 

https://github.com/github/renaming
https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main

could be quite simple just coordinating from longer name `master` to simpler and not de facto standard `main`.